app/src/main/AndroidManifest.xml

```diff
@@ -228,6 +228,12 @@
                 <category android:name="android.intent.category.DEFAULT" />
                 <data android:mimeType="application/*" />
             </intent-filter>
+
+            <intent-filter>
+                <action android:name="android.intent.action.SYNC" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <data android:scheme="file" />
+            </intent-filter>
```

Comment:
you can also move this intent-filter to only debug version manifest 
under app/src/debug create a new `AndroidManifest.xm` and add this to it

```xml
<?xml version="1.0" encoding="utf-8"?><!--
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
        xmlns:tools="http://schemas.android.com/tools"
        package="com.wire.android">

    <application android:name=".WireApplication">
        <activity android:name=".ui.WireActivity">
            <intent-filter>
                <action android:name="android.intent.action.SYNC" />
                <category android:name="android.intent.category.DEFAULT" />
                <data android:scheme="file" />
            </intent-filter>
        </activity>
    </application>
</manifest>

```

then when building gradle will merge the main and the debug files together to create the final app manifest

Reply:
Ah cool! Then I can remove the `if (!BuildConfig.DEBUG)` I assume

Reply:
> Ah cool! Then I can remove the `if (!BuildConfig.DEBUG)` I assume

i would keep it just in case 😅

Reply:
Done